Diagnostics/FreeBusyChecker/Functions/ExoOauthFunctions.ps1

```diff
@@ -0,0 +1,172 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Variables are being used in functions')]
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $false)]
+    [String] $tdEXOIntraOrgConTarGetAddressDomains
+)

```

Comment:
Again, we shouldn't be having this type of a start of a file that we are just importing into a different main script.

Reply:
Mistake, removed